package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "mocha": "mocha --compilers js:babel-register",
```

Comment:
@JohnMadakin is there a reason why `mocha` is added as a script here? Also, I don't think this PR requires `mocha`